mcstatus/address.py

```diff
@@ -57,14 +57,16 @@ class Address(_AddressBase):
         The class is not a part of a Public API, but attributes :attr:`host` and :attr:`port` are a part of Public API.
     """
 
-    def __init__(self, *a, **kw):
+    def __init__(self, host: str, port: int):
         # We don't call super's __init__, because NamedTuples handle everything
         # from __new__ and the passed self already has all of the parameters set.
         self._cached_ip: ipaddress.IPv4Address | ipaddress.IPv6Address | None = None
 
         # Make sure the address is valid
         self._ensure_validity(self.host, self.port)
 
+        super().__init__()
```

Comment:
Is the comment about super's `__init__` above no longer relevant?

Reply:
Forgot to remove it, we should actually be calling init here, it's just that it shouldn't take the attributes.